tests/rpc/test_rpc_telegram.py

```diff
@@ -1458,6 +1458,23 @@ def test_whitelist_static(default_conf, update, mocker) -> None:
     assert ("Using whitelist `['StaticPairList']` with 4 pairs\n"
             "`ETH/BTC, LTC/BTC, XRP/BTC, NEO/BTC`" in msg_mock.call_args_list[0][0][0])
 
+    context = MagicMock()
+    context.args = ['sorted']
+    msg_mock.reset_mock()
+    assert ("Using whitelist `['StaticPairList']` with 4 pairs\n"
+            "`ETH/BTC, LTC/BTC, NEO/BTC, XRP/BTC`" in sorted(msg_mock.call_args_list[0][0][0]))
+
+    context = MagicMock()
+    context.args = ['nobase']
+    msg_mock.reset_mock()
+    assert ("Using whitelist `['StaticPairList']` with 4 pairs\n"
+            "`ETH, LTC, XRP, NEO`" in msg_mock.call_args_list[0][0][0])
+
+    context = MagicMock()
+    context.args = ['nobase', 'sorted']
+    msg_mock.reset_mock()
+    assert ("Using whitelist `['StaticPairList']` with 4 pairs\n"
```

Comment:
You'll have to call the whitelist again.
just resetting the mock - and then asserting won't work properly (and is causing the ci failure)

applies to each of the asserts above.

Reply:
Thanks for this guidance, this is the area I'm least confident with so should be resolved now if it's as simple as I understood it to be.